shim/opentelemetry-opencensus-shim/src/opentelemetry/shim/opencensus/_patch.py

```diff
@@ -0,0 +1,55 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Optional
+
+from opencensus.trace.tracer import Tracer
+from opencensus.trace.tracers.noop_tracer import NoopTracer
+
+from opentelemetry import trace
+from opentelemetry.shim.opencensus._shim_tracer import ShimTracer
+from opentelemetry.shim.opencensus.version import __version__
+
+_logger = logging.getLogger(__name__)
+
+
+def install_shim(
+    tracer_provider: Optional[trace.TracerProvider] = None,
+) -> None:
+    if tracer_provider is None:
+        tracer_provider = trace.get_tracer_provider()
+    otel_tracer = tracer_provider.get_tracer(
+        "opentelemetry-opencensus-shim", __version__
+    )
+    shim_tracer = ShimTracer(NoopTracer(), otel_tracer=otel_tracer)
```

Comment:
So, openCensus sometimes uses noop and sometimes contexttracer. Why do we need to only pass in noop?

Reply:
@jeremydvoss any thoughts?

Reply:
Got it. Thanks